simpleServer.js

```diff
@@ -33,13 +33,13 @@ let getContacts = function(request, response, id) {
     }
 };
 
-let postContacts = function(request, response, id) {
+let postContacts = (request, response, id) => {
     // read in body from browser
     let body = '';
-    request.on('data', function(chunk) {
+    request.on('data', (chunk) => {
         body += chunk.toString();
     });
-    request.on('end', function() {
+    request.on('end', () => {
         let contact = JSON.parse(body);
         contact.id = ++lastId;
         console.log('Created: ' + contact);
```

Comment:
If I understand this right, this code is just adding the new contact but isn't pushing back to the JSON file so it never changes. If not, I would add the functionality here.